packages/patternfly-4/react-core/src/components/Tabs/Tab.tsx

```diff
@@ -22,6 +22,6 @@ export const Tab: React.FunctionComponent<TabProps> = ({
   className = '',
 }: TabProps) => {
   return (
-    null
+    <React.Fragment/>
```

Comment:
Best practice is to return null when there is nothing to render.

"You can return an empty value from a React render method. However return null is the most preferred as it signifies that nothing is returned"

https://stackoverflow.com/questions/42083181/is-it-possible-to-return-empty-in-react-render-function

Reply:
I agree we should return null.

Reply:
Adding the reversion as Nicole is out. 

@redallen Is it possible to update our docgen to detect the Tab properties while maintaining the null return?